apps/web/controllers/repos/index.rb

```diff
@@ -11,15 +11,15 @@ def call(params)
 
     def repos_list(params = {})
       if language = params&.fetch(:language) { nil }
-        return repository.all if language == GitRepo::ALL.downcase
+        return repository.all if language == Repo::ALL.downcase
         repository.by_language(language)
       else
         repository.all
       end
```

Comment:
and also, replace all this logic to interactor/operation/service/etc and inject this object to action with DI. In this case you can use mock object in specs for action unit tests  :)